modules/client-configuration/pages/snippets/addchannels_vendor_webui.adoc

```diff
@@ -1,6 +1,7 @@
 .Procedure: Adding Software Channels
 . In the {productname} {webui}, navigate to menu:Admin[Setup Wizard > Products].
 . Locate the appropriate products for your client operating system and architecture using the search bar, and check the appropriate product.
-    This will automatically check all required channels.
-    Click the arrow to see the complete list of related products, and ensure that any extra products you require are checked.
+  This will automatically check all mandatory channels.
+  Depending on your usecase, also check the optional Client Tools Channel.
+  Click the arrow to see the complete list of related products, and ensure that any extra products you require are checked.
```

Comment:
Sorry, seems I wrote something wrong in the bug report. This happens when you have not really time to deeply review something.

The Client Tools Channels are not "optional". They are marked as "recommended". And recommended channels are also automatically selected as long as you do not uncheck the "include recommended" toggle.

So I wonder if the bugreport is just invalid.

Maybe you can write 

This will automatically check all mandatory channels. Additionally also all recommended channels as long as the "include recommed" toggle is turned on. 
Click the arrow to see the complete list of related products, and ensure that any extra products you require are checked.

Reply:
Not your fault!  Now that I think about, I see that I could have give it a try myself.